---
layout              : page
show_meta           : false
title               : "Resume"
subheadline         : "updated july 2025"
teaser              : "This gives you a brief overview of my professional experiences"
header:
   image_fullwidth  : "header_homepage_13.jpg"
permalink           : "/resume/"
---
Core Competencies:
•		Omics and Data Analysis: scRNA-seq, bulk RNA-seq, WGS (Illumina, PacBio), methylation sequencing, ChIP-seq, ATAC-seq, 3D genomics, Multi-omics data integration
•	Tools & Resources: UK Biobank, All of Us, ClinVar, gnomAD, GTEx, GEO, UCSC Genome Browser
•	Population and Statistical Genetics: GWAS (common and rare), ExWAS, Statistical and functional fine mapping, Heritability, Polygenic risk scores, Multi-ancestry population analysis
•	Data & Programming: Python (pandas, NumPy), R (Seurat, DESeq2), bash, Git, Nextflow, SQL basics
•	Cloud & Computing: HPC environments, Docker, AWS, GCP
WORK EXPERIENCE

FOUNDATION MEDICINE, Boston, MA	05/2024 – 05/2025
Computational Biologist 1
Drove feature discovery and innovation towards the development of a new product as part of a highly interdisciplinary team. 
•	Improved and implemented reproducible pipelines for biomarker discovery using whole-genome methylation sequencing for early cancer detection and tissue-of-origin prediction.
•	Led feature discovery efforts as part of a cross-functional R&D team; collaborated closely with clinical annotation, assay development, and data engineering groups.
•	Contributed to translational product development through analysis of large-scale genomic and clinical datasets.

BOSTON CHILDREN’S HOSPITAL and THE BROAD INSTITUTE, Boston, MA	02/2023 – 05/2024
Postdoctoral Research Fellow
Led independent projects and collaborated with experimental scientists to uncover the mechanistic role of genetic variation on hematopoietic stem cell function. Applied population genomics approaches to quantify the germline factors underlying risk for clonal hematopoiesis and myeloid malignancies.
•	Conducted GWAS and fine mapping in large biobank cohorts (UK Biobank and All of Us) to identify novel genetic risk loci for myeloid malignancies. 
•	Collaborated with experimental biologists to validate computationally generated hypotheses in hematopoietic stem cells.
•	Co-led multi-omics integration analyses (RNA-seq, ChIP-seq, 3D genomics, ATAC-seq) to link genetic variation and 3D genome changes contributing to fetal hemoglobin regulation during erythropoiesis.
•	Harnessed scRNA-seq datasets to uncover changes in transposable element expression in hematopoietic cells.
•	Performed statistical modeling to identify rare ClinVar variants associated with myeloproliferative neoplasms.

TUFTS UNIVERSITY and THE JACKSON LABORATORY, Bar Harbor, ME	06/2017 – 01/2023
Graduate Research Fellow
Designed, executed, and analyzed experiments to uncover the evolutionary patterns of centromere DNA, a repetitive and rapidly evolving region of the genome that is challenging to investigate with existing tools. Combined computational and experimental methods to uncover the scope of diversity at mouse centromeres and the impact of centromere genetic variation on function.
•	Developed customized k-mer based pipelines to quantify genetic and epigenetic sequence variation at diverse mouse centromeres. 
•	Discovered the centromere repeat sequence for the species Mus Pahari and collaborated with an external lab to quantitatively characterize the scope of genetic variation. 
•	Gained expertise in the following experimental techniques: fluorescence in-situ hybridization, fluorescence microscopy, ChIP-seq, CUT&Tag, quantitative-PCR, cell culture, bacterial transfection, western blotting.
•	Collaborated with external labs and mentored junior researchers; awarded NIH F31 fellowship and multiple scientific honors.

EDUCATION

Doctor of Philosophy (PhD), Genetics, TUFTS UNIVERSITY AND THE JACKSON LABORATORY, Bar Harbor, ME
Bachelor of Science (BS), Psychobiology and Marine Biology, UNIVERSITY OF CALIFORNIA, Los Angeles, CA

PUBLICATIONS

Jorge Diego Martin-Rufino*, Alexis Caulier*, …, Uma P. Arora et. al. “Transcription factor networks disproportionately enrich for heritability of blood cell phenotypes” Science, April 2025.
Jiawei Zhao*, Liam D. Cato*, Uma P. Arora et. al. “Inherited blood cancer predisposition through altered transcriptional elongation” Cell, February 2024.
Craig W. Gambogi*, Nootan Pandey*, Jennine M. Dawicki-McKenna*, Uma P. Arora et. al. “Centromere innovations within a mouse species” Science Advances, November 2023.
Uma P. Arora, Beth Sullivan, Beth L. Dumont. “Variation in the CENP-A sequence association landscape across diverse inbred strains” Cell Reports, October 2023.
Uma P. Arora & Beth L. Dumont. “Meiotic drive in house mice: Mechanisms, Consequences, and Insight for Human Biology” Chromosome Research, July 2022.
Raman Akinyanju Lawal, Uma P. Arora, Beth L. Dumont. “Selection shapes the landscape of functional variation in wild house mice” BMC Biology, November 2021.
Uma P. Arora, Caleigh Charlebois, Raman Akinyanju Lawal, Beth L. Dumont. “Population and subspecies diversity at mouse centromere satellites” BMC Genomics, April 2021.

PROFESSIONAL DEVELOPMENT 

Conservation Education Volunteer, NEW ENGLAND AQUARIUM
Postbaccalaureate Student Mentor, THE JACKSON LABORATORY
Carpentries Instructor, THE CARPENTRIES                                                                                       
Ruth L. Kirschstein National Research Service Award F31, NATIONAL CANCER INSTITUTE
DeLill Nasser Award for Professional Development in Genetics, GENETICS SOCIETY OF AMERICA
Lorraine Flaherty Award for Outstanding Oral Presentation, IMGS


1. Open `_config.yml` and work it through, it's well documented.
1. Add your own `logo.png` to `/assets/img/`.
1. Add your own social media links and open `_data/socialmedia.yml`.
1. Open `_sass/_01_settings_colors.scss` and adjust the the colors to your taste.
1. Customize your navigation and open `_data/navigation.yml`.
1. Translate this theme and open `_data/language.yml`.
1. Change the links in the footer and open `_data/services.yml` and `_data/network.yml`.
1. Open `_data/authors.yml` and edit author information and set default author in `config.yml`.
1. Make it yours and add your own content.
1. [Read the documentation][1] to check out all features of *Feeling Responsive*.

<a class="radius button small" href="{{ site.url }}{{ site.baseurl }}/documentation/">Check out the documentation for all the tricks ›</a>


 [1]: {{ site.url }}{{ site.baseurl }}/documentation/
